docker-services.yml

```diff
@@ -109,3 +109,17 @@ services:
       - REDIS_HOSTS=local:cache:6379
     ports:
       - "8081:8081"
+  statping:
+    image: statping/statping:v0.90.74
+    restart: always
+    ports:
+      - "8080:8080"
+    volumes:
```

Comment:
When it doesnt find a `statping.db` or has no permission to create it, it takes you to the setup page. we shouldn't have this, since we provide a `config.yml`
```
    volumes:
      - ./docker/statping/app/config.yml:/app/config.yml
      - ./docker/statping/app/services.yml:/app/services.yml
```